.gitmodules

```diff
@@ -49,7 +49,7 @@
 	url = https://github.com/ceph/seastar.git
 [submodule "src/fmt"]
 	path = src/fmt
-	url = https://github.com/ceph/fmt.git
+	url = https://github.com/fmtlib/fmt.git
```

Comment:
why?

Reply:
That's the part where I don't really understand what I am doing.
I need to bump up the version, but wasn't sure how to achieve that.

Reply:
i created #41925 to partially address the linkage issue of clang + GCC.

Reply:
Thanks